assets/src/stories-editor/deprecations/filters/index.js

```diff
@@ -0,0 +1,3 @@
+export { default as addAMPExtraPropsDeprecations } from './add-amp-extra-props';
+export { default as wrapBlockInGridLayerDeprecations } from './wrap-blocks-in-grid-layer';
+export { default as addAMPAttributesDeprecations } from './add-amp-attributes';
```

Comment:
^ The changes in these filter files aren't actually that big, however, following how the deprecation is done in Gutenberg and also for not making the logic very confusing with different conditions within `helpers.js` file filters, thought that perhaps it makes sense to copy the full filter as it was. Not sure if that's the best approach though, thoughts?

Reply:
Also, applying only some changes conditionally in the filter would mean that the future changes might influence the deprecation logic, too. This would need testing if it's OK in case there's a future deprecation for example.

Anyway -- reducing the repeated code here would definitely be a place for improvement if we can do it considering potential future deprecations + not influencing the deprecation logic with future changes.